SQLQuery/src/main/java/com/oops/app/SQLCloudController.java

```diff
@@ -0,0 +1,42 @@
+package com.oops.app;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+import java.util.Properties;
+
+import javax.sql.DataSource;
+
+public class SQLCloudController {
+
+    private static final String DB_USER = System.getenv("DB_USER");
+    private static final String DB_PASS = System.getenv("DB_PASS");
+    private static final String DB_NAME = System.getenv("DB_NAME");
+
+    private static final String CONNECTION_NAME = System.getenv("CONNECTION_NAME");
+
+
+    //Simple test file
```

Comment:
It's great that you're providing tests

Reply:
Woops technical that is not a test file anymore but all the test are in the test folder